src/views/NewPlantView/index.tsx

```diff
@@ -65,6 +70,41 @@ const NewPlantView = ({ route, navigation }): React.ReactElement => {
     setLoading(false);
   };
 
+  const onCancel = () => {
+    navigation.goBack();
+  };
+
+  const confirmationContent = (plantSelected) => (
+    <View>
+      <Text style={styles.confirmationModalText}>{"Tem certeza que deseja deletar " + plantSelected.name + "?"}</Text>
+      <View style={styles.confirmationModalButtons}>
+        <Button
+          onPress={() => setOpenModal(false)}
+          title="Cancelar"
+          type={buttonTypes.SmallSecondary}
+        /> 
+        <Button
+          onPress={() => removePlant(plantSelected)}
+          title="Confirmar"
+          type={buttonTypes.SmallDanger}
+        />
+      </View>
+    </View>
+  );
+
+  const removePlant = async (plantSelected) => {
+    setLoading(true);
+    await deletePlant(plantSelected._id);
+    navigation.navigate("HomeView");
+    setLoading(false);
+    setOpenModal(false);
```

Comment:
Please verify if make sense the order of this func calls, might better to do the navigation after closing the modal